app/presenters/setting_presenter.rb

```diff
@@ -75,7 +75,7 @@ def category_label
   end
 
   def category_name
-    category.to_s.gsub(/Setting::/, '')
+    category.delete_prefix('Setting::')
```

Comment:
We already save the category as a string in `_add`, and we know which part of the string we need to delete so no need for gsub and regex matching

Reply:
I'm a big fan of `delete_prefix()` (and `delete_suffix()`) since it's so much more expressive in what it achieves.